### ISSUE TITLE
rhash incorrectly outputs "Everything OK" when verifying embedded CRC32 checksums

### ISSUE THREAD

**Issue by bcc32** [rhash/RHash#174]
Reproduction steps:

```sh
touch 'a [00000001]'
touch 'b [00000001]'
rhash -k 'a [00000001]' 'b [00000001]'
```

The rhash command prints out:

```
a [00000001]                                        ERR
b [00000001]                                        ERR
Everything OK
```

While the checksum errors are printed correctly for each file, the "Everything OK" summary is incorrect and confusing.

Version info:

```
rhash --version
RHash v1.4.0
```

**Comment by rhash** [comment]
Thank you for the bug report!

The bug is confirmed for versions 1.3.9-1.4.2 and for the latest sources code (no bug in versions <= 1.3.8).

**Comment by rhash** [comment]
Fixed by 5796c94b1eba4fb7407868555e630b409340b78e.

**Comment by bcc32** [comment]
Thank you!

**Comment by rhash** [comment]
Fix included into [RHash v1.4.3](https://github.com/rhash/RHash/releases/tag/v1.4.3).
